File: src/main/java/domain/Arme.java
package domain;

import java.util.List;

public class Arme {
    private int _id;
    private String name;
    private String level;
    private String imgUrl;
    private List<Statistic> statistics;
    private String type;
    private int setId;
    private List<ArmeCharacteristique> characteristics;

    public static String ARC = "Arc";
    public static String BAGUETTE = "Baguette";
    public static String BATON = "Bâton";
    public static String DAGUE = "Dague";
    public static String EPEE = "Épée";
    public static String FAUX = "Faux";
    public static String MARTEAU = "Marteau";
    public static String PELLE = "Pelle";

    public String getType() {
        return type;
    }
}

File: src/main/java/domain/Arme.kt
package domain

data class Arme(
    val _id: Int,
    val name: String,
    val level: Int,
    val imgUrl: String,
    val statistic: List<Statistic>,
    val type: String,
    val setId: Int,
    val armeCharacteristique: ArmeCharacteristique
)